eng/restore-toolset.ps1

```diff
@@ -10,12 +10,12 @@ function Install-Workloads {
         Write-Output "dotnet is installed at $(Get-Command dotnet)."
     }
 
-    # This workload requires Xcode 15.4
-    dotnet workload install maccatalyst --version 9.0.100-rc.1.24453.3 --source https://api.nuget.org/v3/index.json
-
-    if ($LASTEXITCODE -ne 0) {
-        Write-Output "Failed to install workloads."
-        exit 1
+    try {
+        # This workload requires Xcode 15.4
```

Comment:
Why Xcode 15.4? I'm not sure I understand this requirement

Reply:
Xcode 16 is not yet available in Helix, so it is required to target workload version that supports Xcode 15.4.

.NET 9 rc1 supports Xcode 15.4: https://github.com/dotnet/maui/wiki/Release-Versions